spec/models/reports/facility_state_spec.rb

```diff
@@ -67,4 +59,103 @@
       end
     end
   end
+
+  context "assigned patients by care states" do
+    describe "under_care, lost_to_follow_up, dead" do
+      it "computes the number of assigned patients correctly with each care state" do
+        facility = create(:facility)
+        _dead_patient = create(:patient, recorded_at: june_2021[:long_ago], assigned_facility: facility, status: "dead")
```

Comment:
Mind adding a patient or two who is not diagnosed with HTN, just to be explicit and show that this matview only selects for hypertension?

Reply:
Thanks for the suggestion, Rob. Added in 44db5ab22bc2ae9c15747ab6c66ca13bae0f3689.